test/v3/amendPeriodSummary/AmendPeriodSummaryServiceSpec.scala

```diff
@@ -56,13 +46,6 @@ class AmendPeriodSummaryServiceSpec extends ServiceSpec {
     taxYear = taxYear
   )
 
-  private val requestDataWithCl290Disabled = Def1_AmendPeriodSummaryRequestData(
```

Comment:
This can be kept and changed to use Def2_AmendPeriodSummaryRequestData and Def2_AmendPeriodSummaryRequestBody